config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "admin": 1387103854381760553,
+    "mod": 1387103950099972268,
+    "botDev": 1386724072074772610,
+    "verified": 1386815940791894198,
+    "just_joined": 1389256789513470042,
+    "superUsers": [723377619420184668, 543143780925177857],
+    "muted": 1386947661084495913,
+    "everyone": 0,
+
+    "logs": 0,
+    "modlogs":1386770381733630012,
+    "nqnlogs": 0,
+    "welcomeChannel": 1389645685153857638,
+    "lobbyChannel": 1368604372581486634,
+    "_comment1": "this config is currently configured for testing purposes only"
+}
```

Comment:
Follow [this](https://github.com/sach-12/pesu-bot-verify/blob/app-router/src/utils/config.js#L6) for structuring the config file. Don't need to put all of them, only ones you'd need